tasks/stop-service.yml

```diff
@@ -2,10 +2,15 @@
 - name: Gather service facts
   ansible.builtin.service_facts:
 
+- name: Override jenkins_service_name for non-systemd installations
+  ansible.builtin.set_fact:
+    jenkins_service_name: "jenkins"
+  when: ansible_service_mgr != "systemd"
```

Comment:
Why not have
```yaml
    jenkins_service_name: "jenkins.service"
  when: ansible_service_mgr == "systemd"
```
?
It seems that systemd is the special case.

Reply:
See above.